src/ui/flutter_app/lib/screens/game_page/game_page.dart

```diff
@@ -62,131 +59,93 @@ class _GamePageState extends State<GamePage>
   final Engine _engine = NativeEngine();
 
   double screenPaddingH = AppTheme.boardScreenPaddingH;
-  final double boardMargin = AppTheme.boardMargin;
+  final double _boardMargin = AppTheme.boardMargin;
 
   late String _tip;
-  bool isReady = false;
-  bool isGoingToHistory = false;
-  late Timer timer;
+  bool _isGoingToHistory = false;
   late AnimationController _animationController;
-  late Animation<double> animation;
-  bool disposed = false;
+  late Animation<double> _animation;
+  bool _disposed = false;
   late bool ltr;
   late double boardWidth;
 
   static const String _tag = "[game_page]";
 
-  Future<void> _setReadyState() async {
-    debugPrint("$_tag Check if need to set Ready state...");
-    if (!isReady && mounted) {
-      debugPrint("$_tag Set Ready State...");
-      setState(() {});
-      isReady = true;
-      timer.cancel();
-
-      if (Localizations.localeOf(context).languageCode == "zh" &&
-          !LocalDatabaseService.preferences.isPrivacyPolicyAccepted) {
-        onShowPrivacyDialog();
```

Comment:
Why was the privacy statement dialog box removed? As I mentioned in other posts earlier, according to China’s information security laws, each app must display a privacy statement dialog box when it is first launched.

Reply:
and I mentioned earlier that I didn't remove it just cleaned the code ;)

I fixed it and restructured it :upside_down_face: